the-new-octal/src/app/service/product.service.ts

```diff
@@ -0,0 +1,39 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { environment } from 'src/environments/environment';
+import { Product } from '../model/product';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ProductService {
+
+  apiUrl: string = `${environment.apiUrl}product`;
+  endPoint: string = 'product';
```

Comment:
Egységes elnevezést használjunk!

```suggestion
  entityName: string = 'product';
```

Reply:
Jogos, (bár nem is használtam fel, mivel eleve bedrótoztam a lokális apiURL-be, mostantól viszont újraírva ismét fogom használni.)